package/README.md

```diff
@@ -57,35 +57,42 @@ Here is an overview of the release process. See the section below for detailed i
 
    * When done, put release notes into a private Gist and send the link to other maintainers for review.
 
-   * Update `version` to the release in platform.txt and commit. E.g. `2.5.0`.
+The following points assume work in a direct clone of the repository, and not in a personal fork.
 
-2. Tag the latest commit on the master branch. In this project, tags have form `X.Y.Z`, e.g. `2.4.0`, or `X.Y.Z-rcN` for release versions. Notice that there's no `v`at the beginning of the tag. Tags must be annotated, not lightweight tags. To create a tag, use git command (assuming that the master branch is checked out):
+2. Update `version` to the release in platform.txt and commit. E.g. `2.5.0`. Then git push origin to get the commit merged to master.
+
+3. Tag the latest commit on the master branch. In this project, tags have form `X.Y.Z`, e.g. `2.4.0`, or `X.Y.Z-rcN` for release versions. Notice that there's no `v`at the beginning of the tag. Tags must be annotated, not lightweight tags. To create a tag, use git command (assuming that the master branch is checked out):
```

Comment:
@devyte If you want to keep using `-beta` instead of `rc`, you can update this paragraph. If you, can you please fix "for release versions" to "for release candidate versions"?

Reply:
@igrr we decided to go with -betaX because -rcX was being confused as post-release instead of pre release (e.g. we would receive comments like "I tried 2.4.0 but it crashed, so I updated to 2.4.0-rc2, but then it didn't compile").

What do you think of not doing any more -rcX or -betaX, and just doing normal simple releases? 
- Breaking changes in majors (usually)
- Other changes in minors (avoid breaking changes, except when feature is too broken)
- Majors and minors could be a bit unstable
- Sub releases with stabilization, and critical changes/bug fixes
- Document the linear release model as well as the above guidelines
?